libs/domain/checkout/address/stories/demo.stories.ts

```diff
@@ -0,0 +1,44 @@
+import { resolve } from '@spryker-oryx/di';
+import { AddressService } from '@spryker-oryx/user';
+import {
+  MockAddressService,
+  MockAddressType,
+  mockNormalizedAddresses,
+} from '@spryker-oryx/user/mocks';
+import { Meta, Story } from '@storybook/web-components';
+import { html, TemplateResult } from 'lit';
+import { storybookPrefix } from '../../.constants';
+import { CheckoutAddressOptions } from '../address.model';
+
+export default {
+  title: `${storybookPrefix}/Address`,
+  args: {
+    enableList: false,
+    addressId: mockNormalizedAddresses[0].id,
+  },
+  argTypes: {
+    addressId: {
+      options: mockNormalizedAddresses.map((a) => a.id),
+      control: { type: 'select' },
+      table: { category: 'Demo' },
+    },
+  },
+} as Meta;
+
+interface Props {
```

Comment:
```suggestion
interface Props extends CheckoutAddressOptions {
```

Reply:
I actually use the type at the argument

Reply:
`Story<Props & CheckoutAddressOptions>`

Reply:
Yes, I saw that. But it is more clear and readable then extending on place